homework.js

```diff
@@ -0,0 +1,164 @@
+//summation off 5 function
+const summation = function(sum){
+    
+    
+    let i=0;
+    while (i < 5 )
+    {
+    sum += i;
+    i++;
+    }
+    return sum;
+    
+};
+
+
+// summ Even number 
+  const sumEvn = function (num){
```

Comment:
this code doesnt work because the const inside the loop will keep changing.

this will work

```
 const sumEvn = function (num){
    const sum = 0;

    for (const i = 1; i <= num; i++) {
        if (i % 2 === 0) {
            sum += i;
        }
    }

    return sum;
}
```